### DISCUSSION TITLE
Countdown

### DISCUSSION THREAD

**Discussion by ebussinetz** [liedekef/events-made-easy#624] [Q&A] [answered]
Hi,

we already have a countdown on our homepage but I always have to manually enter the date of the next event.
I wanted to do that with e.g. #_DATETIMEDIFF_TILL_START{d}.
This placeholder gives me a "d".
If I just enter #_DATETIMEDIFF_TILL_START, I get "1 day 16 hours 49 minutes 8 seconds". Firstly, that is much too long and secondly, I would like it in German.
I would prefer it if it looked like this:
https://www.spiritualchor.de
That's why I thought: #_DATETIMEDIFF_TILL_START{D} #_DATETIMEDIFF_TILL_START{H} #_DATETIMEDIFF_TILL_START{I} #_DATETIMEDIFF_TILL_START{S}. But then I get: "D H I S".

Thanks in advance

**Comment by liedekef** [comment, marked as answer]
from the doc https://www.e-dynamics.be/wordpress/event-formatting/ : use php date interval format, so in your case. You missed the part from https://www.php.net/manual/en/dateinterval.format.php where it says "Each format character must be prefixed by a percent sign (%)."

**Reply by liedekef** [reply]
next to that: the default countdown format gets smaller (if the days are 0, it is no longer shown). And also: it is translated per your site language.

**Reply by ebussinetz** [reply]
OK, but the size should allways be the same for this layout. And in my case it was not translatet. The website is set to german.
Thank you for your great support!

**Reply by liedekef** [reply]
Just  tested it by setting my site language to German: "3 Monate 114 Tage 4 Stunden 30 Minuten 33 Sekunden" , seems German to me :-)
Anyway, the provided shortcode is there for your usecase if you need a fixed size anyway :-)
